microprot/scripts/split_search.py

```diff
@@ -529,19 +574,21 @@ def mask_sequence(hhsuite_fp, fullsequence_fp, subsequences_fp=None,
 
     for hit in subseqs_pos:
         _id = get_q_id(hit)
-        header = "%s_%i-%i %s" % (query_id,
-                                  hit['alignment'][_id]['start'],
-                                  hit['alignment'][_id]['end'], query_desc)
+        header = "%s %s" % (correct_header_positions(
+            query_id,
+            hit['alignment'][_id]['start'],
+            hit['alignment'][_id]['end']), query_desc)
```

Comment:
For future considerations : having to traverse through 3 dictionaries always sucks. It may be worthwhile to try to leverage the scikit-bio Sequence API, particularly the metadata handling capabilities.